resources/views/createCourse.blade.php

```diff
@@ -27,46 +27,73 @@
 		<!-- Course semester -->
 		<div class="field">
 			<label for="courseSemester">Course semester</label>
-			<input type="text" name="courseSemester" id="courseSemester" placeholder="Spring/Fall/Summer/Winter and year Ex. Fall 2018" required>
+			<input type="text" name="courseSemester" id="courseSemester" placeholder="Spring/Summer/Fall/Winter and year.  Ex. Fall 2018" title="When is this course taking place?"required>
 		</div>
 		<!-- Course type -->
-		<div class="field">
+		<div class="field" title="A client course is looking to outsource specific tasks. A supplier course is looking for projects to team up with.">
```

Comment:
A lo mejor estos queues tendrían que ser visuales? Tipo un description paragraph?

Reply:
Nice to have, not critical rn.